python/foglamp/common/configuration_manager.py

```diff
@@ -167,14 +167,22 @@ def __init__(self, storage=None):
         # keep_original_items = TRUE keep items in category_val_storage not in category_val_new
         category_val_storage_copy = copy.deepcopy(category_val_storage)
         category_val_new_copy = copy.deepcopy(category_val_new)
+        deprecated_items = []
         for item_name_new, item_val_new in category_val_new_copy.items():
             item_val_storage = category_val_storage_copy.get(item_name_new)
             if item_val_storage is not None:
                 item_val_new['value'] = item_val_storage.get('value')
                 category_val_storage_copy.pop(item_name_new)
+            if "deprecated" in item_val_new:
```

Comment:
and true?

Reply:
before coming to this part, validation checks have already ensured that only 'true' values are available for 'deprecated'.